protos/vehicle_alert.proto

```diff
@@ -19,6 +19,7 @@ enum Audience {
   Unknown = 0;
   Customer = 1;
   Service = 2;
+  Service_fix = 3;
```

Comment:
To be consistent we should use pascal case  `ServiceFix`

Reply:
Resolve this issue in commit b049f55e483cb8ab8576b891387cccedaf97175a.